code/__DEFINES/jobs.dm

```diff
@@ -1,3 +1,11 @@
+
+// At what point do we start enabling more specific jobs in the service
+// department?
+#define MINPOP_JOB_LIMIT 7
+// What do we consider a lowpop roundstart population for the sake
+// of enabling heads of staff
+#define LOWPOP_JOB_LIMIT 14
```

Comment:
```suggestion
// department?
#define MINPOP_JOB_LIMIT 5
// What do we consider a lowpop roundstart population for the sake
// of enabling heads of staff
#define LOWPOP_JOB_LIMIT 12
```

Players who don't ready up will also not count towards these, so we really should be careful to keep them low.